### PR TITLE
fix currency symbol upper case for trocador

### PR DESCRIPTION
**Pull request by vibros68**
This solve the bugs: Unsupported asset type and unmarshal array into Go value of type trocador.Trade on https://github.com/crypto-power/cryptopower/pull/257

### CODE REVIEW

File: instantswap/exchange/trocador/trocador.go [hunk 1]
```diff
@@ -184,10 +184,10 @@ func (t *trocador) CreateOrder(vars instantswap.CreateOrder) (res instantswap.Cr
 		ChargedFee:     0,
 		Destination:    trade.AddressUser,
 		ExchangeRate:   trade.rate(),
-		FromCurrency:   trade.CoinFrom,
+		FromCurrency:   strings.ToUpper(trade.CoinFrom),
```
Comment (dreacot) [inline instantswap/exchange/trocador/trocador.go]: i decided to log the from and to currencies

`FromCurrency: DECRED, ToCurrency: LITECOIN (MAINNET - NO MW)`

while the app checks for DCR and LTC, hence why it is stil showing unsupported asset, we would need to handle this edge case

Comment (dreacot) [reply instantswap/exchange/trocador/trocador.go]: we could pass the coin names through a method that renames them appropriately

Comment (vibros68) [reply instantswap/exchange/trocador/trocador.go]: https://github.com/crypto-power/instantswap/pull/10/commits/5fc1bfb43ecdf495c9b2a43e26f948312aae99d2
Fixed here

Comment (dreacot) [reply instantswap/exchange/trocador/trocador.go]: i'd create a follow up issue for the trocador error message not being clear enough

**Review by dreacot — APPROVED** [review]

**Pull request merged**